src/ILLink.CodeFix/xlf/Resources.cs.xlf

```diff
@@ -0,0 +1,22 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="1.2" xsi:schemaLocation="urn:oasis:names:tc:xliff:document:1.2 xliff-core-1.2-transitional.xsd">
+  <file datatype="xml" source-language="en" target-language="cs" original="../Resources.resx">
+    <body>
+      <trans-unit id="RequiresAssemblyFilesCodeFixTittle">
```

Comment:
We don't have localization so you can probably turn this off (I think there's a property in the analyzer project that's set that does this, something like `<EnableXlfLocalization>false`

Reply:
These files should be deleted.